lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1316,6 +1316,15 @@ def setup_pe_conf(host, hosts, opts={})
                   update_pe_conf(opts[:custom_answers])
                 end
               end
+
+              step "Adding agent profiles" do
```

Comment:
Beaker-Answers should already create agent_platforms if it is generating a pe.conf from scratch: https://github.com/puppetlabs/beaker-answers/blob/master/lib/beaker-answers/pe_conf.rb#L122-L128

And if we are relying on meep to create a meep 2.0 schema, we shouldn't be adding anything to it (beyond the custom_answers that have been placed from the acceptance tests).

I might be missing why this is here though?